mentors/cpp/a.json

```diff
@@ -14,5 +14,13 @@
         "link_url": null,
         "avatar_url": null,
         "bio": "Until now I have worked mainly with C. However, through my master in Embedded Systems I discovered the power of C++. I love the power and the flexibility of this language."
+    },
+    {
+    "github_username": "Alexhans",
```

Comment:
Please fix the indentation. It should be 2 spaces more than the opening bracket `{`